### ISSUE TITLE
Drawer Console Warning: 'new NativeEventEmitter() was called with a non-null argument without the required `removeListeners` method'

### ISSUE THREAD

**Issue by batjko** [react-navigation/react-navigation#9882]
### Current behavior

When launching the app with a drawer navigator, the metro bundler and the emulator show this error:

![image](https://user-images.githubusercontent.com/7287076/131146518-d321dfac-3864-491a-baee-e4cc9a604533.png)

Code snippet:

```jsx
// [...] screen imports

import { createDrawerNavigator } from '@react-navigation/drawer'

const { Navigator, Screen } = createDrawerNavigator()

const DrawerContent = ({ navigation, state }) => { 
  // [...] 
}

export const HomeDrawerNavigator = () => (
  <Navigator
    drawerContent={props => <DrawerContent {...props} />}
    screenOptions={{ headerShown: false }}
  >
    <Screen name="Home" component={BottomTabsNavigator} />
    <Screen name="About" component={AboutScreen} />
    <Screen name="Login" component={LoginScreen} />
    <Screen name="Register" component={RegisterScreen} />
  </Navigator>
)
```

Btw, I tried to recreate in Snack, but it complains about the latest versions of the packages it seems?

### Expected behavior

no warning.

### Reproduction

n/a

### Platform

- [X] Android
- [ ] iOS
- [ ] Web
- [ ] Windows
- [ ] MacOS

### Packages

- [ ] @react-navigation/bottom-tabs
- [X] @react-navigation/drawer
- [ ] @react-navigation/material-bottom-tabs
- [ ] @react-navigation/material-top-tabs
- [ ] @react-navigation/stack
- [ ] @react-navigation/native-stack

### Environment

| package                                | version |
| -------------------------------------- | ------- |
| @react-navigation/native               | 6.0.2
| @react-navigation/bottom-tabs          | 6.0.5
| @react-navigation/drawer               | 6.1.4
| @react-navigation/material-bottom-tabs | -
| @react-navigation/material-top-tabs    | -
| @react-navigation/stack                | 6.0.7
| @react-navigation/native-stack         | 6.1.0
| react-native-safe-area-context         | 3.3.0
| react-native-screens                   | 3.6.0
| react-native-gesture-handler           | 1.10.3
| react-native-reanimated                | 1.13.3
| react-native-tab-view                  | 3.1.1
| react-native-pager-view                | -
| react-native                           | 0.65.1
| expo                                   | -
| node                                   | 16.8
| npm or yarn                            | npm v7.21.0

**Comment by tarunnurat** [comment]
hi, I'm still getting this issue. Any workaround?

**Comment by djwesty** [comment]
Same here

**Comment by quyle1222** [comment]
me too

**Comment by zachbarnes** [comment]
I was able to fix this by upgrading react-native-reanimated to 2.2.1.

https://github.com/facebook/react-native/issues/32037#issuecomment-916944027

**Comment by muzhaqi16** [comment]
I have `"react-native-reanimated": "^2.2.2",` and I am still getting the same issue.

**Comment by andresribeiro** [comment]
@muzhaqi16 me too

**Comment by jrhager84** [comment]
Any word on this? I'm getting the same issue. RN .66, reaniamted 2.2.4, drawer  6.1.8

**Comment by thanveershah** [comment]
I am facing the same issue

**Comment by alienriquebm** [comment]
Hi! I have the same problem here, also, the warning is showing up only on Andorid devices and not iOS devices... any news about this?

**Comment by MuhammadRafeh** [comment]
I am also getting the same error!

**Comment by StephenCavender** [comment]
tldr: I resolved this issue by uninstalling and reinstalling the app on my iOS simulator.

I installed the drawer navigator per docs after I'd been developing my app and testing it out on the iOS simulator for a couple weeks. After installing I ran into this issue. Restarting metro did not help. Running `pod install` did not help. I fired up an Android emulator to see if I had the same issue there. I had not yet installed my app on my Android emulator. The app ran fine with the Drawer Navigator. This got me thinking I should try a complete reinstall of the app on my iOS simulator. It worked.

**Comment by roboticambiental** [comment]
I resolved this issue by adding to babel.config.js this:
  module.exports = {
      ...
      plugins: [
          ...
          'react-native-reanimated/plugin',
      ],
  };

source:
https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/installation/

npm start -- --reset-cache

and reinstall the App

All of this with the following dependencies versions:
"dependencies": {
    "@react-navigation/drawer": "^6.4.2",
    "@react-navigation/native": "^6.0.10",
    "@react-navigation/stack": "^6.2.1",
    "react": "18.0.0",
    "react-native": "0.69.0",
    "react-native-gesture-handler": "^2.5.0",
    "react-native-reanimated": "^2.8.0",
    "react-native-safe-area-context": "^4.3.1",
    "react-native-screens": "^3.13.1"
  },

**Comment by waqaskhanroghani** [comment]
use this version it's solved my issue:  "react-native-reanimated": "2.3.0-alpha.2",
